docker/README.md

```diff
@@ -3,46 +3,86 @@
 
-If you just want to have an Orion Context Broker running as fast as possible jump to section [The Fastest Way](#the_fastest_way).
+If you just want to have an Orion Context Broker running as fast as possible jump to section *The Fastest Way*.
 
-If you want to know what and how we are doing things with the container step by step you can  go ahead and read the [Build](#build_the_image) and [Run](#run_the_container) sections.
+If you want to know what and how we are doing things with the container step by step you can go ahead and read the build and run sections.
 
 ## The Fastest Way
 
-A Docker Compose file is provided for convenience. You must install [Docker Compose](https://docs.docker.com/compose/install/) for this method to work.
+Docker Compose lets you have an Orion Context Broker container along with a MongoDB container running in a few minutes. You must install [Docker Compose](https://docs.docker.com/compose/install/) for this method to work.
 
-Simply navigate to the docker directory of the fiware-orion code (if you have downloaded it) and run
+### Run a container from an image you just built
 
-	docker-compose up
+If you have downloaded the [Orion's code](https://github.com/telefonicaid/fiware-orion/) simply navigate to the docker directory and run
 
-If you haven't or you don't want to download the whole thing, you can download the file called `docker-compose.yaml` in a directory of your choice and run the aforementioned command. It will work just the same.
+	sudo docker-compose up
+
+You are using this [docker-compose.yml](docker-compose.yml) file. This will instruct to build a new image for Orion and run it linking it to another MongoDB container.
+
+### Run a container pulling an image from the cloud (recommended)
+
+If you haven't or you don't want to download the Orion repository, you can create a file called `docker-compose.yml` in a directory of your choice and fill it with the following content
+
+	orion:
+	  image: fiware/orion
+	  links:
+	    - mongo
+	  ports:
+	    - "1026:1026"
+	  command: -dbhost mongo
+
+	mongo:
+	  image: mongo:2.6
+
+Then run
+
+	sudo docker-compose up
+
+This way is equivalent to the previous one, except that it pulls the image from the Docker Registry instead of building your own. Keep in mind though that everything is run locally. 
+
+> **Warning**
+> Everything you do with Orion Context Broker when dockerized is non-persistent. *You will lose all your data* if you turn off the MongoDB container. This happens with either this method or the manual one.
+> If you want to prevent this from happening take a look at [this link](https://registry.hub.docker.com/_/mongo/) in section *Where to Store Data* of the MongoDB docker documentation. In it you will find instructions and ideas on how to make your MongoDB data persistent.
 
```

Comment:
probably you can say the same with less words ... 

Everything you do with Orion Context Broker when dockerized is non-persistent. *You will lose all your data* if you turn off the MongoDB container. This happens with either this method or the manual one. [Here((https://registry.hub.docker.com/_/mongo/) you can find how to make ....